packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -33,7 +33,7 @@ import { makeTracer } from '../makeTracer.js';
 
 const { quote: q, details: X } = assert;
 
-const trace = makeTracer('XykAmm');
+const trace = makeTracer('XykAmm', false);
```

Comment:
If the motivation is for the git history, this PR has a separate tiny commit for the tracer changes. [d3a8dc9](https://github.com/Agoric/agoric-sdk/pull/5538/commits/d3a8dc9931d1551e08cf5c591a7212c48103c8ec)

If it's for a different reason, lemme know.

Reply:
Yes, the motivation is for the git history and blame attribution.  How does a separate commit make a difference? Were you planning to not squish when merging? There are 6 commits currently, so would that mean squishing everything else and leaving one commit separate? that doesn't seem sustainable for reviewers or PR authors. Wouldn't it be much simpler to have a separate tiny PR?

Reply:
> Were you planning to not squish when merging? 

Correct. That's why I've been careful to separate independent changes into different commits.

> Wouldn't it be much simpler to have a separate tiny PR?

It would be more work on top of what I've already done to separate the commits. It would be simpler in that we wouldn't have this debate on this PR but in the long run it means more work in general if we can't allow a PR to have multiple independent commits merged. It's worth getting aligned as an org on what is expected and allowed. (I say allowed because the request here was made as a blocking review, such that if someone else approves it the PR still can't be merged).

The CONTRIBUTING.md says,
> We favor "Squash and merge" (which requires the PR title to be a conventional commit) since it requires the least careful attention, especially when there are appended follow-up changes, but it's not the only tool in the toolbox.
> …
> We've also squashed follow-up commits manually (through an interactive rebase) after a PR had been approved and tests were passing, then landed the stack with "Rebase and merge" or "Create merge commit".

The latter is what I've done here.